frappe/docs/user/fr/guides/app-development/trigger-event-on-deletion-of-grid-row.md

```diff
@@ -1,22 +1,45 @@
 
```

Comment:
Seems you've modified the file in the French documentation folder? I'm not sure why the previous documentation was in English in the first place. Can you move it?